module/VuFind/src/VuFind/Export.php

```diff
@@ -71,32 +58,33 @@ class Export
     /**
      * Constructor
      *
-     * @param Config $mainConfig   Main VuFind configuration
-     * @param Config $exportConfig Export-specific configuration
+     * @param array       $mainConfig   Main VuFind configuration
+     * @param array       $exportConfig Export-specific configuration
+     * @param PhpRenderer $viewRenderer View renderer
```

Comment:
Renderer added here because it will be needed in another method as well in the upcoming Zotero Web Library export.